### DISCUSSION TITLE
TypeError: 'nick'

### DISCUSSION THREAD

**Discussion by AAxolotl1** [TwitchIO/TwitchIO#437] [Questions]
```python
bot = twitchio.Client(
    token=TWITCH_TOKEN,
    nick=TWITCH_USERNAME,
    initial_channels=[TWITCH_CHANNEL]
)
``` 

I keep getting a TypeError for nick and I'm not quite sure why. I'm relatively new to coding so I'm not sure exactly what tricks I should try to fix it but I made sure twitchio is installed correctly and that the variables used are typed correctly (in case the variables were causing it). Help or any ideas is greatly appreciated.

Full Error:
```python
Traceback (most recent call last):
  File "(File Location)", line 12, in <module>
    bot = twitchio.Client(
          ^^^^^^^^^^^^^^^^
TypeError: Client.__init__() got an unexpected keyword argument 'nick'
```

**Comment by IAmTomahawkx** [comment]
You seem to be following an outdated tutorial for twitchio. Client no longer takes `nick` as an argument. simply remove it and that error should disappear.
